docs/ExpectAPI.md

```diff
@@ -890,6 +890,17 @@ test('the best drink for octopus flavor is undefined', () => {
 
 You could write `expect(bestDrinkForFlavor('octopus')).toBe(undefined)`, but it's better practice to avoid referring to `undefined` directly in your code.
 
+### `.toBeNan()`
```

Comment:
```suggestion
### `.toBeNaN()`
```

Reply:
👍

Reply:
This still says `Nan` here and in the versioned docs, did you forget commit/push the change?

Reply:
Fixed 👍